src/components/organization/OrganizationForm/OrganizationForm.tsx

```diff
@@ -114,7 +114,10 @@ export function OrganizationForm(props: OrganizationFormProps) {
                         ? `Mise à jour effectuée`
                         : `Organisation créée`,
                     message: props.organization ? (
-                        <>La mise à jour a bien été effectuée</>
+                        <>
+                            La mise à jour a bien été effectuée. Elle sera
```

Comment:
```suggestion
                            La modification sera
```
(pour ne pas se répéter)